draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -228,13 +227,9 @@ bit are:
 
 - the offset up to which the sender commits to sending might be smaller than
   Final Size,
-- this offset might get reduced by subsequent RESET_STREAM_AT frames,
-- the closure is accompanied by an error code, and
-- the RESET_STREAM_AT frame does not contain any payload like the STREAM frame
```

Comment:
I'm not against removing this line, though I might ask removing "only" from the sentence above. Otherwise, people might complain, because this is going to be a different on the logic the sender side.

Reply:
Fair point. I changed it to "main differences".